.github/CODEOWNERS

```diff
@@ -5,7 +5,6 @@
 #*       @defunkt
 * @NaureenBharwaniNOAA @christinaholtNOAA @elcarpenterNOAA @maddenp-noaa @weirae
```

Comment:
I believe we'll still be able to merge this anyway. Once Paul and Brian are in the org, this should be fine.